### PR TITLE
Lithuanian, Latvian, Estonian banks added

### PR DESCRIPTION
**Pull request by s-limo**

### CODE REVIEW

**Comment by andrewpaliga** [comment]
@trigras please take a look at our prescribed format:

> 38 by 24 pixels
Solid white background

The logos you uploaded will not be readable by buyers on our checkout. We can only accept payment icons in the format above (there should be examples in the `payment_icons` folder too). Ideally, there should be at most three letters in the icon (if any) or else they won't be readable.

**Comment by trigras** [comment]
Overlooked size and background - should be fixed now

Regarding readability - it is really complicated. Sometimes one can better recognize "unreadable" icon by shape and colors  then three letter icon. From other hand - there are guidelines how use companies logos and in most cases they do not allow to cut out any part of them. I tried my best to update icons to they could be easily recognized by users. Please let me know if you still think there are unacceptable icons.

File: db/payment_icons.yml [hunk 1]
```diff
@@ -103,3 +103,39 @@
   label: NetBanking
   group: bank_transfers
 
+
```
Comment (andrewpaliga) [inline db/payment_icons.yml]: I think you have an extra white space here.

Comment (trigras) [inline db/payment_icons.yml]: fixed, and it looks "-" was missing as separators

**Comment by millingab** [comment]
@s-limo 
Before I merge this PR, make sure these two steps are done.
1. Make the background of Paysera, Citadele, LHV, Nordea and SEB icons to white
2. Remove the DNB.svg file if it is not suppose to be added in payment_icons.yml

**Comment by andrewpaliga** [comment]
👍

**Pull request merged**